__tests__/components/JellyForm.spec.js

```diff
@@ -0,0 +1,396 @@
+/* globals expect, describe, it */
```

Comment:
Would this component deprecate the `Form` in the future or would both of them live their happy separate lives ?
The reason I ask is because, if I'm reading this correctly, a lot of the tests from here could be shared with `Form`, if we still want to invest in `Form` ?

Reply:
@cyplo They're not separate - the `JellyForm` implementation wraps the `Form` component offering a higher level of abstraction. They indeed share a lot of test cases and could possible be deduped, but I much prefer verbosity over dryness in testing.